src/app/ngrx/reducers/podcast.reducer.ts

```diff
@@ -41,6 +41,7 @@ export function PodcastReducer(state: PodcastState = initialState, action: AllAc
     case ActionTypes.CMS_PODCASTS: {
       return {
         ...state,
+        error: null,
```

Comment:
Was having a problem where the "You are not logged in" message wasn't being cleared, when we retried loading the Podcasts.